devops/resources/cloudflare_purge_cache.sh

```diff
@@ -13,17 +12,25 @@ assume-role ${ROLE_ARN}
 
 aws s3 ls s3://${BUCKET} --recursive | awk '{print $4}' > targets
 
+if [[ "${SITE}" = "https://edx.org" ]]; then
+ZONE_ID=${EDX_ZONE_ID}
+elif [[ "${SITE}" = "https://edx-cdn.org" ]]; then
+ZONE_ID=${EDX_CDN_ZONE_ID}
+elif [[ "${SITE}" = "https://edx-video.net" ]]; then
+ZONE_ID=${EDX_VIDEO_ZONE_ID}
+fi
+
 if ${CONFIRM_PURGE}; then
 python util/cloudflare/by_origin_purger/purger.py\
-    --cloudflare_zone_id ${ZONE_ID}\
+    --cloudflare_zone_id $ZONE_ID\
```

Comment:
```suggestion
    --cloudflare_zone_id ${ZONE_ID}\
```
It's a good practice to have the curly braces on variables in shell scripts. Please keep them.